pages/administration/tools/ticket-lookup.vue

```diff
@@ -78,12 +78,11 @@ watch(hardwareScannedDetails.isInvalid, (newVal) => {
 });
 
 function onInvalidCode() {
-  errorToast.fire({
-    title: 'Invalid ticket QR code scanned'
-  });
+  errorToast.fire({ title: 'Invalid ticket QR code scanned' });
 }
 
 async function onScan(ticketData: TicketQRCodeData) {
+  console.log('Scanned data:', ticketData);
```

Comment:
Did you mean to leave this in the production code? I assume it's not a security problem, but want to make sure it's conscious.